src/Common/src/System/Net/NegotiationInfoClass.cs

```diff
@@ -11,5 +11,6 @@ internal partial class NegotiationInfoClass
         internal const string NTLM = "NTLM";
         internal const string Kerberos = "Kerberos";
         internal const string Negotiate = "Negotiate";
+        internal const string Basic = "Basic";
```

Comment:
Basic is not used in negotiatestream. Seems like this is a partial class with the other part of implementation in windows Interop. For maintaining structure, I think I would define these locally to HttpListener and not reuse the Common ones.